<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QrkGastroQuickProduct</class>
 <widget class="QDialog" name="QrkGastroQuickProduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Artikelname</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>MwSt.</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Preis</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="productLineEdit"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="taxComboBox"/>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="grossLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QrkPushButton" name="okPushButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>Ãœbernehmen</string>
         </property>
         <property name="icon">
          <iconset resource="../../qrk.qrc">
           <normaloff>:/src/icons/ok.png</normaloff>:/src/icons/ok.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QrkPushButton" name="cancelPushButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>Abbrechen</string>
         </property>
         <property name="icon">
          <iconset resource="../../qrk.qrc">
           <normaloff>:/src/icons/cancel.png</normaloff>:/src/icons/cancel.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QrkPushButton</class>
   <extends>QPushButton</extends>
   <header>qrkpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>productLineEdit</tabstop>
  <tabstop>taxComboBox</tabstop>
  <tabstop>grossLineEdit</tabstop>
  <tabstop>okPushButton</tabstop>
  <tabstop>cancelPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../qrk.qrc"/>
  <include location="../../qrk.qrc"/>
 </resources>
 <connections/>
</ui>
